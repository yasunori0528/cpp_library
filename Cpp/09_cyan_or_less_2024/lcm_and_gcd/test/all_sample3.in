4
161749 170324 185833 197173
