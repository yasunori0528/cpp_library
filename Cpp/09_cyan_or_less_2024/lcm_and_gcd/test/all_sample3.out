1 1 49 633795587
